src/test/java/com/fasterxml/jackson/databind/deser/JsonIdentityInfoIdProperty3838Test.java

```diff
@@ -0,0 +1,197 @@
+package com.fasterxml.jackson.databind.deser;
```

Comment:
Needs to go under `...deser.objectid` but I can move it after merge if necessary.

Reply:
Seems to be done by [this commit](https://github.com/FasterXML/jackson-databind/commit/998ca54978c3bcedc90398c2303acffa2f6ff63c#diff-0e6a39d906c028115b3ceb46d470ec298533705bd940973e6a8b5a7803abe2d0), but in `...objectid` packge. Moving on.

Reply:
yes. And you are right, no "deser" in there, my bad.